omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.html

```diff
@@ -38,7 +38,12 @@
     <tbody>
         <tr ng-repeat="row in $ctrl.output.rows">
             <td ng-repeat="cell in row" ng-class="cell.className">
-                {{ cell.translate ? (cell.value | translate) : cell.value }}
+                <span ng-if="!$ctrl.isArray(cell)">
+                    {{ cell.translate ? (cell.value | translate) : cell.value }}
+                </span>
+                <span ng-if="$ctrl.isArray(cell)" ng-repeat="cellElem in cell">
```

Comment:
It looks from the code and the tests like `cell` will never be an array. So this can be simplified, which is nice.

```suggestion
                <span ng-repeat="cellElem in cell">
```

Reply:
The cells containing the obs will always be an array but, a cell can also contain the encounter date and, if configured, the encounter type. Those will not be an array.
I guess we'll need to cope with both situations when iterating.

Reply:
Oh you're right! Yep.